scalastyle-config.xml

```diff
@@ -60,8 +61,8 @@ This file is divided into 3 sections:
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
- * limitations under the License.
- */]]></parameter>
+ * limitations under the License.\E[\w\W]*\Q
+ */\E]]></parameter>
```

Comment:
I don't think the licence exactly specifies 'where' it needs to be. I will check the licence and if possible move it to class comment. You also mentioned you had other examples where you did this so I will check those too.